src/Pim/Bundle/ApiBundle/tests/integration/Controller/Product/CreateProductIntegration.php

```diff
@@ -881,7 +881,7 @@ public function testResponseWhenProductAlreadyExists()
             'errors'  => [
                 [
                     'property' => 'identifier',
-                    'message'  => 'The value simple is already set on another product for the unique attribute sku',
+                    'message'  => 'The same identifier is already set on another product',
```

Comment:
it's ok, we accept to change message on the API (even if we should avoid, but in this case, the message is cleaner like that)